lte/gateway/c/core/oai/test/amf/test_amf_stateless.cpp

```diff
@@ -708,6 +709,10 @@ TEST_F(AMFAppStatelessTest, TestAfterRegistrationComplete) {
                                                sizeof(ue_smc_response_hexbuf));
   EXPECT_EQ(rc, RETURNok);
 
+  s6a_update_location_ans_t ula_ans = util_amf_send_s6a_ula(imsi);
+  rc = amf_handle_s6a_update_location_ans(&ula_ans);
+  EXPECT_TRUE(rc == RETURNok);
```

Comment:
Use Expect_EQ wherever possible so that you can get an informative error message in case of failure.
```suggestion
  EXPECT_EQ(rc, RETURNok);
```

Reply:
addressed